mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot

```diff
@@ -0,0 +1,107 @@
+*** Settings ***
+Resource        ${KVM_RESOURCE}
+
+Suite Setup     Suite Setup
+
+
+*** Variables ***
+${T}    ${CURDIR}
```

Comment:
Why are you renaming `CURDIR` to `T`? (if that's what you're doing)

Reply:
Just to keep it shorter below…

Reply:
Ok :joy: Just wanted to check that there's not a deep, robot reason here that I'm missing.